### DISCUSSION TITLE
Kong Ingress controller error - "context deadline exceeded"

### DISCUSSION THREAD

**Discussion by anup-krai** [Kong/kong#8964] [Help] [answered]
Hi,
We are using Kong with KIC deployed in K8s cluster with Postgres DB.
Kong - 2.7.1
KIC - 2.2.1
We are continuously seeing below error in KIC logs -
```
time="2022-06-16T10:55:05Z" level=error msg="could not update kong admin" error="2 errors occurred:\n\tfailed to sync all entities: context deadline exceeded\n\twhile processing event: {Update} upstream inventory.443.svc failed: making HTTP request: Put \"https://localhost:8444/upstreams/01158245-1730-4c79-a294-c67001d062fb\": context deadline exceeded\n" subsystem=proxy-cache-resolver
```
Because of this error new objects (ingress, services etc) are not getting deployed in Kong.
How can we resolve this error?

**Comment by hanshuebner** [comment]
Hello @anup-krai,

this question might better be asked in the [KIC](https://github.com/Kong/kubernetes-ingress-controller) repository as the error seems to be coming from KIC, not Kong Gateway.

Kind regards,
Hans

**Comment by anup-krai** [comment, marked as answer]
Issue is resolved after adding `proxy_timeout_seconds` environment variable in KIC. Thanks.
